src/test/java/com/github/dockerjava/cmd/CreateContainerCmdIT.java

```diff
@@ -488,14 +485,15 @@ public void createContainerWithDevices() throws DockerException {
 
     @Test
     public void createContainerWithPortBindings() throws DockerException {
+        int baseport = dockerRule.getKind().equals("jersey")? 11000: 12000;
```

Comment:
getKind() is string value, test itself is parametrized with enum, so you can do it easy with `getFactoryType() == JERSEY`